### PR TITLE
dev-libs/libfilezilla: version bump to 0.21.0

### PR DESCRIPTION
**Pull request by xp-vit**
Signed-off-by: Viktar Patotski xp.vit.blr@gmail.com

### CODE REVIEW

File: dev-libs/libfilezilla/libfilezilla-0.21.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+EAPI=7
+inherit flag-o-matic
+
+DESCRIPTION="C++ library offering some basic functionality for platform-independent programs"
+HOMEPAGE="https://lib.filezilla-project.org/"
+SRC_URI="https://download.filezilla-project.org/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0/4" # libfilezilla .so version
```
Comment (thesamesam) [inline dev-libs/libfilezilla/libfilezilla-0.21.0.ebuild:12]: Please bump SONAME: https://abi-laboratory.pro/?view=timeline&l=libfilezilla

**Comment by voyageur** [comment]
Thanks for the PR, I hoped to get https://bugs.gentoo.org/show_bug.cgi?id=717736 addressed before bumping, but limited time... Once SONAME is fixed we can probably pull this in and bump filezilla in separate one (with dependencies current versions will not use this one anyway)

**Comment by xp-vit** [comment]
> Thanks for the PR, I hoped to get https://bugs.gentoo.org/show_bug.cgi?id=717736 addressed before bumping, but limited time... Once SONAME is fixed we can probably pull this in and bump filezilla in separate one (with dependencies current versions will not use this one anyway)

I'm currently testing this package with ebuildtester and it hangs for me on tests like this:
2020-05-07 12:23:19,802 - a939f7 (stdout): make  check-TESTS
2020-05-07 12:23:19,804 - a939f7 (stdout): make[2]: Entering directory '/var/tmp/portage/dev-libs/libfilezilla-0.21.0/work/libfilezilla-0.21.0/tests'
2020-05-07 12:23:19,812 - a939f7 (stdout): make[3]: Entering directory '/var/tmp/portage/dev-libs/libfilezilla-0.21.0/work/libfilezilla-0.21.0/tests'
2020-05-07 12:23:27,819 - a939f7 (stdout): PASS: ratelimit_test

And it hangs on this line forever... version 0.20.1 installs correctly.
That's why I have put PR into draft, trying to understand why it can be happening.. Do you have any ideas on how to proceed with investigation?

**Comment by voyageur** [comment]
Maybe something stricter in ebuildtester env? Locally it seems good:
```
make[3]: Entering directory '/var/tmp/portage/dev-libs/libfilezilla-0.21.0/work/libfilezilla-0.21.0/tests'
PASS: ratelimit_test
PASS: test
============================================================================
Testsuite summary for libfilezilla 0.21.0
============================================================================
# TOTAL: 2
# PASS:  2
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
make[3]: Leaving directory '/var/tmp/portage/dev-libs/libfilezilla-0.21.0/work/libfilezilla-0.21.0/tests'
```

**Comment by xp-vit** [comment]
@voyageur Then let's bump it. thanks for double-checking!

**Comment by voyageur** [comment]
In progress, I just changed SLOT and will push it with a few other bumps in progress

**Pull request closed without merging**